test/js/length-value.js

```diff
@@ -1,15 +1,15 @@
 suite('LengthValue', function() {
-  test('fromValue returns instanceof a LengthValue and StyleValue', function() {
-    var fromVal = (new LengthValue()).fromValue(9.2, 'px');
-    assert.instanceOf(fromVal, SimpleLength, 'A new fromValue should be an instance of SimpleLength');
-    assert.instanceOf(fromVal, LengthValue, 'A new fromValue should be an instance of LengthValue');
-    assert.instanceOf(fromVal, StyleValue, 'A new fromValue should be an instance of StyleValue');
+  test('fromDictionary returns instanceof a LengthValue and StyleValue', function() {
+    var fromDictionary = (new LengthValue()).fromValue(9.2, 'px');
```

Comment:
Also, you should be able to just write

LengthValue.fromValue(9.2, 'px')

instead of also creating a LengthValue then calling the static method on it.